cron/monthly/send-monthly-report.js

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node
+
+process.env.PORT = 3066;
+
+import _ from 'lodash';
+import app  from '../../index';
+import moment from 'moment';
+import config from 'config';
+
+const emailLib = require('../../server/lib/email')(app);
+const debug = require('debug')('monthlyreport');
+
+const Group = app.set('models').Group;
+const User = app.set('models').User;
+const Notification = app.set('models').Notification;
+
+const processGroups = (groups) => {
+    const promises = [];
+    groups.map(g => {
+        promises.push(processGroup(g));
+    })
+    return Promise.all(promises);
+};
+
+const d = new Date;
+d.setMonth(d.getMonth() - 1);
+const month = moment(d).format('MMMM');
+
+const init = () => {
+
+  const query = {
+      attributes: [
+          'id',
+          'slug',
+          'name',
+          'currency'
+      ],
+      where: {
+        slug: { $in: ['railsgirlsatl'] } // for testing
```

Comment:
Should this be in here? It would be better to do this through the notification table too. That way, we can easily enable for a group or two and then extend. Once it works, we can automatically add it for new groups to the table.